troubleshooting/tools/log-loss-test-framework/awslogs-benchmarking-scripts/ec2_launcher.sh

```diff
@@ -0,0 +1,55 @@
+export TASK_DEF_FILE=task-def.json
+export RUN_TASK_FILE=ec2-run-task.json
+export TOTAL_SIZE_IN_MB="1000"
+export THROUGHPUTS="
+1000
+2000
+3000
+4000
+4500
+5000
+6000
+7000
+8000
+9000
+10000
+12000
+"
+export BUFFER_SIZES="
+1m
```

Comment:
it better have a comment to tell what is the unit for the buffer size , 1MB ?

Reply:
added